readme.md

```diff
@@ -1010,3 +1010,13 @@ platformBrowserDynamic(providers)
     .bootstrapModule(AppModule)
     .catch(err => console.error(err));
 ```
+## Injecting strategies into Angular components
+
+It is important to note that the @Strategy decorator cannot be used directly within the constructor of an Angular component. Angular is not aware of the @Strategy decorator so it is unable to resolve the strategies at runtime. When working with Angular components the Injector API should be used instead. This can be accessed by importing the `getRootInjector` function.
+
+```typescript
+import { getRootInjector } from '@morgan-stanley/needle';
+
+const strategies = getRootInjector().getStrategies('work-strategies');
+
+```
```

Comment:
I think we need to provide more here.  Its not just strategies.  its all the decorator types so `optional`, `factory`  `token`.  

Further they do not have to use getRootInjector().  In the case of the component construction say, they should use `Injector` which can be injected into a component.  This is more testable and will also take into account scopes.

Reply:
Update docs to apply to all decorators with matching example